config/azure/setBuildVars.py

```diff
@@ -85,7 +85,7 @@
 print( "Setting $(Cortex.Build.Name) to %s" % buildName )
 print( "##vso[task.setvariable variable=Cortex.Build.Name;]%s" % buildName )
 
-dependencies = "./dependencies"
+dependencies = "./dependencies" if formatVars["platform"] != "windows" else "dependencies"
```

Comment:
I _think_ linux should work fine without the `./` too

Reply:
Taken care of in a925db1